.travis.yml

```diff
@@ -4,6 +4,7 @@ php:
   - 5.4
   - 5.5
   - 5.6
+  - '7'
```

Comment:
@lorenzo That's what travis [docs](http://docs.travis-ci.com/user/languages/php/#PHP-7) show.